app/lib/submission_builder/ty2024/states/md/documents/md502_su.rb

```diff
@@ -11,10 +11,13 @@ class Md502Su < SubmissionBuilder::Document
             def document
               build_xml_doc("Form502SU", documentId: "Form502SU") do |xml|
                 xml.Subtractions do |subtractions|
-                  if calculated_fields.fetch(:MD502_SU_LINE_AB).positive?
-                    subtractions.OtherDetail do |other_detail|
-                      other_detail.Code "AB"
-                      other_detail.Amount calculated_fields.fetch(:MD502_SU_LINE_AB)
+                  Efile::Md::Md502SuCalculator::CODE_LETTERS.each do |code_letter|
+                    code_letter_calculation = "MD502_SU_LINE_#{code_letter}".to_sym
+                    if calculated_fields.fetch(code_letter_calculation).positive?
```

Comment:
I've seen elsewhere in the XML checking if the `reitirement_ui` flag is enabled, & then showing these fields, but given that there will not be any follow up data without the UI flag enabled, this will not show up to begin with since the value for the letter will not be positive.